optaplanner-docs/src/main/asciidoc/Chapter-Partitioned_search/Chapter-Partitioned_search.adoc

```diff
@@ -8,27 +8,214 @@
 :sourcedir: .
 
 
-[[partitionedSearchOverview]]
-== Overview
+[[partitionedSearchAlgorithm]]
+== Algorithm Description
 
-For very big datasets, it is sometimes worthwhile to partition the datasets into smaller pieces.
+For big datasets, usually above 5000 planning entities,
+it is usually worthwhile to partition the datasets into smaller pieces and solve those separately.
+Partition Search is multi-threaded, leading to a performance boost due to more CPU utilization.
+Additionally, it finds an initial solution faster, even if it only uses only 1 CPU,
+because the search space sum of a partitioned Construction Heuristic is far less than its non-partitioned variant.
```

Comment:
changed to "Additionally, even when only using one CPU, it finds an initial solution faster,
because the search space sum of a partitioned Construction Heuristic is far less than its non-partitioned variant.
"